Model/Config/Backend/V2enable.php

```diff
@@ -0,0 +1,54 @@
+<?php
+/*
+ * Copyright Extend (c) 2023. All rights reserved.
+ * See Extend-COPYING.txt for license details.
+ */
+
+namespace Extend\Integration\Model\Config\Backend;
+
+use Magento\Framework\App\Config\Value;
+use Extend\Integration\Setup\Model\AttributeSetInstaller;
+use Extend\Integration\Setup\Model\ProductInstaller;
+
+class V2enable extends Value
+{
+    private AttributeSetInstaller $attributeSetInstaller;
+    private ProductInstaller $productInstaller;
+
+    public function __construct(
+        \Magento\Framework\Model\Context $context,
+        \Magento\Framework\Registry $registry,
+        \Magento\Framework\App\Config\ScopeConfigInterface $config,
+        \Magento\Framework\App\Cache\TypeListInterface $cacheTypeList,
+        AttributeSetInstaller $attributeSetInstaller,
+        ProductInstaller $productInstaller,
+        \Magento\Framework\Model\ResourceModel\AbstractResource $resource = null,
+        \Magento\Framework\Data\Collection\AbstractDb $resourceCollection = null
+    ) {
+        parent::__construct(
+            $context,
+            $registry,
+            $config,
+            $cacheTypeList,
+            $resource,
+            $resourceCollection
+        );
+        $this->attributeSetInstaller = $attributeSetInstaller;
+        $this->productInstaller = $productInstaller;
+    }
+
+    /**
+     * This will create a new Product Protection product when V2 PP is enabled
+     *
+     * @return V2enable
+     */
+    public function afterSave()
+    {
+        $value = (int) $this->getValue();
+        if ($value === 1) {
```

Comment:
What's the significance of 1 here? Maybe we want to create an explicitly named constant, a comment would probably suffice too.

Reply:
It's just a true/false value, basically copied the block as it exists in V1enable. It can probably technically get away with the name 'value' because it's a class extending 'Value' and is representing a primitive type. I'm on board with your suggestion of giving it an explicit name so let's do that.